bin/cli.js

```diff
@@ -166,10 +163,35 @@ const runNode = (config, cb) => {
 };
 
 const runBrowser = (config, cb) => {
-  const server = new karma.Server(config.browser, cb);
+  const buildDir = path.resolve('./build');
+  const buildAdapter = path.resolve('./build/adapter.js');
+  const adapter = path.resolve(__dirname, '../karma/adapter.js');
+
+  if (!fs.existsSync(buildDir)) fs.mkdirSync(buildDir);
+  const data = fs.readFileSync(adapter);
+  const required = config.files
+    .map(file => `require('../${file}');`)
+    .join('');
+
+  fs.writeFileSync(buildAdapter, data + required);
+  const server = new karma.Server(config.browser, code => {
+    fs.unlinkSync(buildAdapter);
+    fs.rmdirSync(buildDir);
+    cb(code);
+  });
   server.start();
 };
 
+const runBoth = (config, cb) => {
+  runBrowser(config, codeB => {
```

Comment:
Maybe just run them in parallel, woulldn't they be independent?

Reply:
Output from node and browser will mix

Reply:
Oh, really, ok.